.vscode/settings.json

```diff
@@ -10,7 +10,7 @@
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
     "files.trimTrailingWhitespace": true,
-    "editor.tabSize": 4,
+    "editor.tabSize": 2,
```

Comment:
This is fine with me but I would then recommend to set python linter to 2 spaces/tab as well ... this way everything is coherent

Reply:
I personally would prefer 4 ... but having 2 in the linter and 4 in vscode is not ideal